homeassistant/components/plum_lightpad/__init__.py

```diff
@@ -61,8 +61,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Plum Lightpad from a config entry."""
     _LOGGER.debug("Setting up config entry with ID = %s", entry.unique_id)
 
-    username = entry.data.get(CONF_USERNAME)
-    password = entry.data.get(CONF_PASSWORD)
+    username = entry.data[CONF_USERNAME]
+    password = entry.data[CONF_PASSWORD]
```

Comment:
These are compulsory according to the config_entry schema, and `await load_plum(username, password, hass)` doesn't allow `None` for username and password